lib/genesis_collector/collector.rb

```diff
@@ -129,7 +129,7 @@ def safely
 
     def shellout_with_timeout(command, timeout = 2)
       response = `timeout #{timeout} #{command}`
-      unless $CHILD_STATUS.success?
+      if $CHILD_STATUS.exitstatus == 124  # timeout exits 124 on timeout
```

Comment:
I wonder if we were counting on the old behavior anywhere. This is more accurate so it should be ok but something to keep an eye out for.